benchmarks/perturbations.py

```diff
@@ -67,5 +69,6 @@ def setup(self):
         self.rho0 = Earth.rho0.to(u.kg / u.km**3).value  # kg/km^3
```

Comment:
```suggestion
        self.rho0 = rho0_earth.to_value(u.kg / u.km ** 3)
```

and this comes from `from poliastro.constants import rho0_earth`